core/src/main/scala/org/apache/spark/ui/exec/ExecutorsPage.scala

```diff
@@ -26,7 +26,7 @@ import org.apache.spark.ui.{ToolTips, UIUtils, WebUIPage}
 import org.apache.spark.util.Utils
 
 /** Summary information about an executor to display in the UI. */
-private case class ExecutorSummaryInfo(
+private[ui] case class ExecutorSummaryInfo(
```

Comment:
It turns out that this is somehow necessary for the MiMa checks to pass.  We should add a comment to explain this.